dataframe-excel/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/xlsx.kt

```diff
@@ -195,6 +218,29 @@ public fun DataFrame.Companion.readExcel(
     return dataFrameOf(columns)
 }
 
+/**
+ * This is a universal function for name repairing
+ * and should be moved to the API module later,
+ * when the functionality will be enabled for all IO sources.
+ */
+private fun repairNameIfRequired(nameFromCell: String, columnNameCounters: MutableMap<String, Int>, nameRepairStrategy: NameRepairStrategy): String {
+    return when(nameRepairStrategy) {
```

Comment:
linting

Reply:
There's no space after `when`, same with the `if`s a bit below. No clue why the linter is not picking those up.

Reply:
ok, will check twice, thank you